cernopendata/config.py

```diff
@@ -162,8 +162,10 @@
 RECORDS_REST_FACETS = {
```

Comment:
This would have been better in a PR of its own, since it is fully orthogonal to "markdown for articles"...